src/container/srv_container.c

```diff
@@ -3531,6 +3531,16 @@ upgrade_cont_cb(daos_handle_t ih, d_iov_t *key, d_iov_t *val, void *varg)
 		return rc;
 	}
 
+	entry = daos_prop_entry_get(prop, DAOS_PROP_CO_GLOBAL_VERSION);
```

Comment:
I think this block needs to follow the cont_prop_read() call since prop is NULL here.